qutebrowser/browser/downloads.py

```diff
@@ -695,12 +718,11 @@ def fetch(self, reply, fileobj=None, filename=None, auto_remove=False):
         """
         if fileobj is not None and filename is not None:
             raise TypeError("Only one of fileobj/filename may be given!")
-        if filename is not None:
-            suggested_filename = os.path.basename(filename)
-        elif fileobj is not None and getattr(fileobj, 'name', None):
-            suggested_filename = fileobj.name
-        else:
-            _inline, suggested_filename = http.parse_content_disposition(reply)
+        if not suggested_filename:
+            if filename is not None:
+                suggested_filename = os.path.basename(filename)
+            elif fileobj is not None and getattr(fileobj, 'name', None):
+                suggested_filename = fileobj.name
```

Comment:
Yeah, removed it because the only place it was called did `http.parse_content_dixposition` anyway. But I agree that it makes sense to have it there.